atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -336,6 +336,9 @@ def get_odcs_config(self):
             odcs_config = ODCSConfig(**odcs_config_kwargs)
         return odcs_config
 
+    def config_not_empty(self):
```

Comment:
This is an odd name for it, how about `is_empty` ?

Reply:
sure